workspace/apps/pidp/src/scss/_pidp-variables.scss

```diff
@@ -0,0 +1,15 @@
+// This file is duplicated at libs\shared\ui\src\scss\_pidp-variables.scss.
+// All changes to this file should be replicated in the duplicate file above.
+// If a method exists to share a scss file between libraries, that should be implemented
+// so the duplicates of this file will no longer need to be kept in sync.
```

Comment:
To reduce duplication and avoid maintaining two files, we can import this partial `pidp-variables.scss` into `app.scss` and then it would be available for use in any file in `/libs` as well.